streamalert_cli/terraform/monitoring.py

```diff
@@ -72,7 +72,7 @@ def generate_monitoring(cluster_name, cluster_dict, config):
 
     if monitoring_config.get('kinesis_alarms_enabled', True):
         cluster_dict['module']['cloudwatch_monitoring_{}'.format(cluster_name)].update({
-            'kinesis_stream': '{}_{}_stream_alert_kinesis'.format(prefix, cluster_name),
+            'kinesis_stream': '${{module.kinesis_{}.stream_name}}'.format(cluster_name),
```

Comment:
Should we keep `prefix` here?

Reply:
@chunyong-lin this does effectively still use the prefix - except it uses the output from here:

https://github.com/airbnb/streamalert/pull/1027/files/9f738ef03c371190e9540fc0cdecd04c9df10fd5#diff-517d65d9de13bb32b0329474e9b11627R6

which gets its stream name from here:

https://github.com/airbnb/streamalert/pull/1027/files/9f738ef03c371190e9540fc0cdecd04c9df10fd5#diff-916958fe1956202328ac3949bb5ea39cR33

Reply:
this change avoids defining the "stream name" multiple places.. potentially missing one if the other changes